thrift_tools/thrift_struct.py

```diff
@@ -0,0 +1,60 @@
+
+class ThriftStruct(object):
+    """A thrift struct"""
+
+    def __init__(self, fields):
+        self._fields = fields
+
+    @property
+    def fields(self):
+        return self._fields
+
+    def __eq__(self, other):
+        return (isinstance(other, self.__class__)
+                and self.__dict__ == other.__dict__)
+
+    def __len__(self):
+        return len(self._fields)
+
+    def __getitem__(self, key):
+        return self._fields[key]
+
+    def __iter__(self):
+        return iter(self._fields)
+
+
+class ThriftField(object):
```

Comment:
I just made a commit which moved some methods to thrift_struct class , I am new to namedtuples (& python) but I assume since now the class has some real behavior & is no longer just a placeholder for few attributes - it can't be a namedtuple. Did I get this right?